hack/k8s/helm/nuclio/values.yaml

```diff
@@ -136,9 +136,13 @@ registry: {}
     # username: someuser
     # password: somepass
 
-  #  Use dedicated base/onbuild images registry (pull registry)
+  #  Use dedicated onbuild images registry (pull registry)
   #
-  # defaultBaseRegistryURL: someurl
+  # defaultBaseRegistryURL: someUrl
+
+  # Use this registry url as an override for both base and onbuild images,
+  # so they will be pulled from the given registry URL and not from the default registry their name implies
```

Comment:
I suggest changing the description comment to the following:
```
  # URL of a registry for pulling both base and "onbuild" images, instead of
  # pulling from the default registry that's implied from the image names
```

Reply:
Fixed to what we agreed on